schema/GenomeAnnotationEnum.mcf

```diff
@@ -1131,28 +1134,30 @@ typeOf: dcs:DNASequenceRoleEnum
 descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help"
 description: "The object represents a chromosome, linkage-group, or plasmid."
 
-Node: dcid:DNASequenceRoleUnlocalized-scaffold
-name: "DNASequenceRoleUnlocalized-scaffold"
+Node: dcid:DNASequenceRoleUnlocalizedScaffold
+name: "DNASequenceRoleUnlocalizedScaffold"
 typeOf: dcs:DNASequenceRoleEnum
 descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help"
 description: "The object is a scaffold that is associated with a particular chromosome but has not been localized to a specific position on the chromosome."
 
-Node: dcid:DNASequenceRoleUnplaced-scaffold
-name: "DNASequenceRoleUnplaced-scaffold"
+Node: dcid:DNASequenceRoleUnplacedScaffold
+name: "DNASequenceRoleUnplacedScaffold"
 typeOf: dcs:DNASequenceRoleEnum
 descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help"
 description: "The object is a scaffold that does not have a chromosome assignment."
 
-Node: dcid:DNASequenceRoleAlt-scaffold
+Node: dcid:DNASequenceRoleAltScaffold
 name: "DNASequenceRoleAlt-scaffold"
 typeOf: dcs:DNASequenceRoleEnum
 descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/help"
 description: "The object is either an alternate loci scaffold or a patch scaffold."
 
-Node: dcid:DNASequenceRoleFix-patch
+Node: dcid:DNASequenceRoleFixPatch
 name: "DNASequenceRoleFix-patch"
 typeOf: dcs:DNASequenceRoleEnum
+description: "The object is a fix patch to the assembly-unit scaffold."
 
-Node: dcid:DNASequenceRoleNovel-patch
+Node: dcid:DNASequenceRoleNovelPatch
```

Comment:
a few more places where we should match dcid with name. (This would cause the schema-checker to fail)